packages/lesswrong/server/scripts/swapUserEmails.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-console */
+
+import Users from "../collections/users/collection";
+
+export const swapUserEmails = async (userId1: string, userId2: string) => {
+  console.log(`Fetching users: ${userId2}, ${userId2}`);
+  const [user1, user2] = await Promise.all([
+    Users.findOne({_id: userId1}),
+    Users.findOne({_id: userId2}),
+  ]);
+  if (!user1 || !user2) {
+    throw new Error("Invalid users");
+  }
+  console.log(`Fetched users: ${user1.displayName}, ${user2.displayName}`);
+  await Promise.all([
+    Users.rawUpdateOne({_id: user1._id}, {
+      $set: {
+        email: user2.email,
+        emails: user2.emails,
+        "services.auth0": user2.services.auth0,
+      },
+    }),
+    Users.rawUpdateOne({_id: user2._id}, {
+      $set: {
+        email: user1.email,
+        emails: user1.emails,
+        "services.auth0": user1.services.auth0,
+      },
+    }),
+  ]);
```

Comment:
It could be overkill, but would you mind putting this in a transaction (I'm not sure what the best way to go about this is, maybe to getSqlClient and then do this via raw sql queries)? I thought there was a unique constraint on email addresses, which would cause this to fail. It seems like there isn't (there should be!), but either way these fields are prone to having constraints put on them and are annoying to recover